src/index.js

```diff
@@ -0,0 +1,46 @@
+import './style.css';
+
+const lists = document.querySelector('[data-lists]');
+
+const todoArr = [
+  {
+    index: 1,
+    description: 'Buy nuts',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Complete USP',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: 'Read the Deep Work',
+    completed: false,
+  },
+];
+const displayToDo = () => {
+  const displayToDo = todoArr.map(
+    (item) => `
+    <div class="todo item">
+    <input type="checkbox" class="checkbox" />
+    <p>${item.description}</p>
+    <span class="delete" data-id="${item.id}">X</span>
+  </div> `,
+  );
+  lists.innerHTML = displayToDo.join(' ');
+};
```

Comment:
- [ ] @elhamatokhi a great job on your app, however when the index is changed the todo items are not displayed according to the index as shown below, kindly implement this feature as its part of the requirement
![image](https://user-images.githubusercontent.com/61408860/213726640-6d993ab0-c77c-4364-849f-5f3c2cffc920.png)
![image](https://user-images.githubusercontent.com/61408860/213726801-5deb1b1b-7fc6-4a6a-98fe-bfee1764b8cc.png)

Reply:
Thank you for the kind review. Would you kindly make it a little bit clear? As I did not understand the main requirement. I have traversed the items using their index value.